PROJECT_CHARTER.md

```diff
@@ -0,0 +1,51 @@
+# `jest` Charter
+
+Jest is a modular test runner and framework for JavaScript, designed for running using Node.js on all of its supported platforms. It is a modular runner than can run and test any JavaScript, TypeScript, or other language that compiles to JavaScript. It is designed to be easy to use and to have a low barrier to entry, while still being flexible and powerful.
+
+## Section 0: Guiding Principles
+
+The `jest` project is part of the [OpenJS Foundation][openjs foundation], which operates transparently, openly, collaboratively, and ethically. Project proposals, timelines, and status must not merely be open, but also easily visible to outsiders.
```

Comment:
```suggestion
The `jest` project is part of the [OpenJS Foundation][openjs foundation]. The project operates transparently, openly, collaboratively, and ethically. Project proposals, timelines, and status must not merely be open, but also easily visible to outsiders.
```
Make the guiding principles about the project, not the foundation (the mixup comes from the time when node was both the project and the foundation.